website/addons/wiki/static/wikiPage.js

```diff
@@ -166,6 +166,27 @@ function ViewModel(options){
     self.viewVis = ko.observable(options.viewVisible);
     self.compareVis = ko.observable(options.compareVisible);
     self.menuVis = ko.observable(options.menuVisible);
+    // singleVis : checks if the item visible is the only visible column
+    self.singleVis = ko.computed(function(){
+        var visible = 0;
+        var single = undefined;
+        if(self.editVis()){
+            visible++;
+            single = 'edit';
+        }
+        if(self.viewVis()){
+            visible++;
+            single = 'view';
+        }
+        if(self.compareVis()){
+            visible++;
+            single = 'compare'
+        }
+        if(visible === 1){
+            return single;
+        }
+        return undefined;
```

Comment:
Returning `undefined` is unintuitive. `undefined` is the value for properties that aren't assigned. That's not true here; `singleVis()` *is* defined. Perhaps return `null` or `false`.